initializr-service/src/main/java/io/spring/initializr/service/InitializrService.java

```diff
@@ -68,4 +75,24 @@ public Executor getAsyncExecutor() {
 
 	}
 
+	@Bean
+	public Filter hstsFilter() {
+		return new HttpHeaderSecurityFilter();
+	}
+
+	@Bean
+	public TomcatServletWebServerFactory tomcatServletWebServerFactory() {
```

Comment:
Isn't there a way to do that with a customizer?

Reply:
I guess we could do it with a `TomcatContextCustomizer`. Is there an advantage to doing that over this?

Reply:
It feels wrong to create the whole factory for something that is an add-on to the auto-config. The advantage here is that Spring Boot is still in control (rather than us taking over which isn't really what we should do here IMO)

Reply:
ok, I'll update it